### ISSUE TITLE
使用Masonry setupAutoHeightWithBottomView 自适应Cell高度时无法成功, 有方案适配吗?

### ISSUE THREAD

**Issue by wumenghua** [tigerAndBull/TABAnimated#128]
### New Issue Checklist

* [ ] I have read the [Documentation](https://github.com/tigerAndBull/TABAnimated/tree/master/Documents)
* [ ] I have searched for a similar issue in the [project](https://github.com/rs/TABAnimated/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. iOS / macOS
 Platform Version        | e.g. 12.0 / 10.14.0
 TABAnimated Version      | e.g. 5.0.0 / 4.4.0
 Xcode Version           | e.g. Xcode 11 / Xcode 10
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by tigerAndBull** [comment]
需要设置高度，demo的场景你看下能满足吗


------------------ 原始邮件 ------------------
发件人: fennuxiong <notifications@github.com&gt;
发送时间: 2020年12月17日 19:00
收件人: tigerAndBull/TABAnimated <TABAnimated@noreply.github.com&gt;
抄送: Subscribed <subscribed@noreply.github.com&gt;
主题: 回复：[tigerAndBull/TABAnimated] 使用Masonry setupAutoHeightWithBottomView 自适应Cell高度时无法成功, 有方案适配吗? (#128)

**Comment by wumenghua** [comment]
dome不能满足.  自适应高度能否通过传入模拟数据计算cell高度后在绘制骨架图, 这需要程序做修改才行; 
目前方案是新建固定高度Cell(模仿自适应Cell的UI)来显示骨架图, 不过会增加一些编码和维护时间;

**Comment by tigerAndBull** [comment]
是可以这么做的，后者方案是权宜之计，前者我需要你提供测试样例

**Comment by tigerAndBull** [comment]
前者也可以做到不修改源程序，但是可能需要适配，可能你用法不对，总之你需要提供下场景
